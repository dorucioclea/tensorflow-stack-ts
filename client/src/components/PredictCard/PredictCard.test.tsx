// Jest snapshot test
import * as React from "react";
import * as TestRenderer from "react-test-renderer";
import { PredictCard } from ".";

test("jest works", () => {
    expect(1).toBeTruthy();
});

describe("PredictCard testing", () => {

    test("PredictCard snapshot is ok", () => {
        const component = TestRenderer.create(
            <PredictCard />,
        );
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});

