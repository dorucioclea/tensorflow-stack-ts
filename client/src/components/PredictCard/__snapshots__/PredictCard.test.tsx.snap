// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PredictCard testing PredictCard snapshot is ok 1`] = `
<div
  className="ant-card ant-card-bordered"
>
  <div
    className="ant-card-head"
    style={Object {}}
  >
    <div
      className="ant-card-head-wrapper"
    >
      <div
        className="ant-card-head-title"
      >
        Server Predict
      </div>
    </div>
  </div>
  <div
    className="ant-card-body"
    style={Object {}}
  >
    <div
      className="ant-col-10"
      style={undefined}
    >
      <span
        className="ant-input-group ant-input-group-compact"
        onBlur={undefined}
        onFocus={undefined}
        onMouseEnter={undefined}
        onMouseLeave={undefined}
        style={undefined}
      >
        Input: 
        <textarea
          className="ant-input server-predict-input"
          onChange={[Function]}
          onKeyDown={[Function]}
          style={Object {}}
          value="[[1,2,3,5,8,13,21,34,55,89]]"
        />
        <button
          className="ant-btn ant-btn-primary"
          onClick={[Function]}
          title={undefined}
          type="button"
        >
          <span>
            Predict
          </span>
          <i
            className="anticon anticon-right"
          >
            <svg
              aria-hidden="true"
              className=""
              data-icon="right"
              fill="currentColor"
              height="1em"
              onClick={undefined}
              style={undefined}
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M765.7 486.8L314.9 134.7A7.97 7.97 0 0 0 302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 0 0 0-50.4z"
              />
            </svg>
          </i>
        </button>
      </span>
    </div>
    <div
      className="ant-col-1"
      style={undefined}
    />
    <div
      className="ant-col-10"
      style={undefined}
    >
      <span
        className="ant-input-group"
        onBlur={undefined}
        onFocus={undefined}
        onMouseEnter={undefined}
        onMouseLeave={undefined}
        style={undefined}
      >
        Prediction:
        <textarea
          className="ant-input server-predict-output"
          onChange={[Function]}
          onKeyDown={[Function]}
          style={Object {}}
          value=""
        />
        <br />
        <button
          className="ant-btn disabled ant-btn-primary"
          onClick={[Function]}
          title={undefined}
          type="button"
        >
          <span>
            Graph it
          </span>
          <i
            className="anticon anticon-down"
          >
            <svg
              aria-hidden="true"
              className=""
              data-icon="down"
              fill="currentColor"
              height="1em"
              onClick={undefined}
              style={undefined}
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
              />
            </svg>
          </i>
        </button>
        <button
          className="ant-btn  disabled ant-btn-ghost"
          onClick={[Function]}
          title={undefined}
          type="button"
        >
          <span>
            Clear graph
          </span>
          <i
            className="anticon anticon-delete"
          >
            <svg
              aria-hidden="true"
              className=""
              data-icon="delete"
              fill="currentColor"
              height="1em"
              onClick={undefined}
              style={undefined}
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M360 184h-8c4.4 0 8-3.6 8-8v8h304v-8c0 4.4 3.6 8 8 8h-8v72h72v-80c0-35.3-28.7-64-64-64H352c-35.3 0-64 28.7-64 64v80h72v-72zm504 72H160c-17.7 0-32 14.3-32 32v32c0 4.4 3.6 8 8 8h60.4l24.7 523c1.6 34.1 29.8 61 63.9 61h454c34.2 0 62.3-26.8 63.9-61l24.7-523H888c4.4 0 8-3.6 8-8v-32c0-17.7-14.3-32-32-32zM731.3 840H292.7l-24.2-512h487l-24.2 512z"
              />
            </svg>
          </i>
        </button>
      </span>
    </div>
    <div
      className="ant-divider ant-divider-horizontal"
    />
    <div
      className="ant-col-21"
      style={undefined}
    >
      <div
        className=""
        style={undefined}
      />
      <p>
        Hover-on for point tooltip; Click and drag to pan view; Scroll to zoom in/out
      </p>
    </div>
  </div>
</div>
`;
